src/Temperature-Accretech_UFseries/main.py

```diff
@@ -113,26 +119,46 @@ def connect(self) -> None:
         #     self.device_communication[self.instance_key] = self.port
 
         self.prober = accretech_uf.AccretechProber(self.port)
-
         self.prober.set_verbose(self.verbosemode)
 
+        self.unique_identifier = "Driver_Accretech_UFseries_" + self.port_str
```

Comment:
It's important that the Temperature and WaferProber driver use the same unique_identifier. Maybe it's worth adding a comment that mentions this fact. Just in case someone wants to change something in the future so they are directly informed to also change it for the other affected driver.